### ISSUE TITLE
Change max-len to add option to ignore text within p tags

### ISSUE THREAD

**Issue by oguru** [eslint/eslint#14253]
**What rule do you want to change?**

max-len

**Does this change cause the rule to produce more or fewer warnings?**

Fewer

**How will the change be implemented? (New option, new default behavior, etc.)?**

New option for ignoring text inside html p tags

**Please provide some example code that this change will affect:**

```
<p>It is for comparison purposes only and does not accurately represent the layout of available plots.</p>
```

**What does the rule currently do for this code?**

"This line has a length of 121. Maximum allowed is 80."

**What will the rule do after it's changed?**

Remove the error

**Are you willing to submit a pull request to implement this change?**

I'm not sure I have the ability

**Comment by mdjermanovic** [comment]
Hi @oguru, thanks for the issue! 

The team has made the difficult decision to freeze stylistic rules (please find more details in [this blog post](https://eslint.org/blog/2020/05/changes-to-rules-policies)), so we won't be adding any new options to the `max-len` rule.

You can use the generic [ignorePattern](https://eslint.org/docs/rules/max-len#ignorepattern) option to ignore lines that match a regular expression:

```js
/* eslint max-len: ["error", { ignorePattern: "<p>" }] */

<p>It is for comparison purposes only and does not accurately represent the layout of available plots.</p>
```

[Online Demo](https://eslint.org/demo#eyJ0ZXh0IjoiLyogZXNsaW50IG1heC1sZW46IFtcImVycm9yXCIsIHsgaWdub3JlUGF0dGVybjogXCI8cD5cIiB9XSAqL1xuXG48cD5JdCBpcyBmb3IgY29tcGFyaXNvbiBwdXJwb3NlcyBvbmx5IGFuZCBkb2VzIG5vdCBhY2N1cmF0ZWx5IHJlcHJlc2VudCB0aGUgbGF5b3V0IG9mIGF2YWlsYWJsZSBwbG90cy48L3A+Iiwib3B0aW9ucyI6eyJwYXJzZXJPcHRpb25zIjp7ImVjbWFWZXJzaW9uIjoxMiwic291cmNlVHlwZSI6InNjcmlwdCIsImVjbWFGZWF0dXJlcyI6eyJqc3giOnRydWV9fSwicnVsZXMiOnt9LCJlbnYiOnt9fX0=)


I'm closing this issue per our [rules policies](https://github.com/eslint/eslint#stylistic-rule-updates).

**Comment by oguru** [comment]
Aah, that will work. Not sure why I didn't think about that! Sorry for wasting your time and thanks for the example.
